openupgrade_scripts/scripts/account/14.0.1.1/post-migration.py

```diff
@@ -399,14 +399,23 @@ def _create_fixed_vietnam_bank_accounts(env):
             [("chart_template_id", "=", vn_chart_template.id)]
         )
         for company in companies:
+            Account = env["account.account"]
+            random_account = Account.search([("company_id", "=", company.id)], limit=1)
+            digits = len(random_account.code) if random_account else 6
+            liquidity_account_prefix = company.bank_account_code_prefix or ""
+            default_account_code = Account._search_new_account_code(
+                company, digits, liquidity_account_prefix
+            )
```

Comment:
@tranngocson1996 như này thôi cho ngắn
```suggestion
            default_account_code = Account._search_new_account_code(
                company,
                company.chart_template_id.code_digits,
                company.bank_account_code_prefix or "",
            )
```